src/components/atoms/EmailPrintButton.test.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+
+import { Email, Print } from "./EmailPrintButton.stories";
+
+it("renders EmailPrintButton in its Email state", () => {
+  render(<Email {...Email.args} />);
+  const text = screen.getByText("Email");
+  expect(text).toBeTruthy();
+
+  const icon = document.getElementsByTagName("span")[0];
```

Comment:
probably better to set data-testid on the span element rather than getting it by its tag name. However this is a minor comment

Reply:
Added data-testid for the span element